docs/_tutorials/phpqa-tutorial.md

```diff
@@ -30,6 +30,17 @@ Start PHPTesting!!!
 
 ## How to use it?
 
+### Create a git repository
+
+The `phpqa` tool expects you to be operating in the context of a `git`
+repository. So before continuing, you'll want to decide where you'll be storing
+your test files, and create a `git` repo in that directory.
```

Comment:
You don't need a `git` repository. You can store your `.phpt` files anywhere in your system. For example, you can simply create a `.phpt` file in your $HOME folder and run `phpqa run $HOME/your_test.phpt` and it should work. If this is not working as expected for you could you please open an issue at the [docker-phpqa](https://github.com/herdphp/docker-phpqa) repository?